### PR TITLE
Order endpoints

### PR DESCRIPTION
**Pull request by BrunoMilitoFrancois**

### CODE REVIEW

File: ZPastel.API/Controllers/OrderController.cs [hunk 1]
```diff
@@ -10,18 +15,49 @@ namespace ZPastel.API.Controllers
...
         [Produces("application/json")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public async Task<ActionResult> CreateOrder()
+        public async Task<ActionResult> CreateOrder(OrderResource orderResource)
         {
+            var order = orderConverter.ConvertToModel(orderResource);
+
+            await orderService.CreateOrder(order);
+
             return NoContent();
```
Comment (joseluz) [inline ZPastel.API/Controllers/OrderController.cs]: Create a task to return the Created Order

File: ZPastel.API/Filters/ExceptionFilter.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        {
+            this.logger = logger;
+            this.hostEnvironment = hostEnvironment;
+        }
+
+        public override void OnException(ExceptionContext context)
+        {
+            var exception = context.Exception;
+
+            switch (exception)
```
Comment (joseluz) [inline ZPastel.API/Filters/ExceptionFilter.cs]: No switch case

File: ZPastel.API/Controllers/OrderController.cs (hunk 1 above)
Comment (BrunoMilitoFrancois) [reply ZPastel.API/Controllers/OrderController.cs]: Created Task to return the created order

File: ZPastel.API/Filters/ExceptionFilter.cs (hunk 2 above)
Comment (BrunoMilitoFrancois) [reply ZPastel.API/Filters/ExceptionFilter.cs]: removed switch case

**Pull request merged**